src/AssetPack/AssetPack.router.ts

```diff
@@ -298,21 +308,54 @@ export class AssetPackRouter extends Router {
     return uploader.single(THUMBNAIL_FILE_NAME)
   }
 
-  private retrieveDefaultAssetPacks = async (): Promise<
-    FullAssetPackAttributes[]
-  > => {
+  private checkAndUpdateDefaultAssetPacksCache = async () => {
+    const currentTimestamp = Date.now()
     const aDayPassed =
-      Date.now() - this.lastDefaultAssetPacksFetch >
+      currentTimestamp - this.lastDefaultAssetPacksFetch >
       Number(DEFAULT_ASSET_PACK_CACHE) // 24 * 60 * 1000
 
     if (this.defaultAssetPacks.length === 0 || aDayPassed) {
       const defaultAssetPacks = await AssetPack.findByEthAddressWithAssets(
         DEFAULT_ETH_ADDRESS
       )
       this.defaultAssetPacks = this.sanitize(defaultAssetPacks)
-      this.lastDefaultAssetPacksFetch = Date.now()
+      this.defaultAssetPacksCachedResponse = JSON.stringify({
+        ok: true,
+        data: this.defaultAssetPacks,
+      })
+      this.lastDefaultAssetPacksFetch = currentTimestamp
     }
+  }
 
+  private sendDefaultAssetPacksRaw = async (
+    res: express.Response,
+    tracer: string
+  ) => {
+    const defaultAssetPacksRaw = await this.logExecutionTime(
+      this.getDefaultAssetPacksRaw,
+      'Get default asset packs for default owner',
+      tracer
+    )
+    res.setHeader('Content-Type', 'application/json')
+    res.status(200).send(defaultAssetPacksRaw)
```

Comment:
A bit of a stretch, but maybe we can take advantage of STATUS_CODES and use `STATUS_CODES.ok` here too (or just delete the `status()` call altogether)

Reply:
Removed the status call!